src/AlertCard/AlertCard.story.mdx

```diff
@@ -0,0 +1,207 @@
+      expanded content toast variant.
+    </AlertCard>
+  </Story>
+</Preview>
+
+## Warning Alert
+
+Warning alerts present information about the action that is about to be taken that alerts the user to consequences of the action and asks for confirmation that the action still be taken in light of the consequences.
+
+<Preview>
+  <Story name="warning light">
+    <AlertCard
+      onClose={() => undefined}
+      color={colors.orange.base}
+      icon={<IconWarningSolid />}
+      heading="Warning alert"
+      actions={<Button size="small">Confirm</Button>}
+    >
+      exceededSubLimitAt is a deprecated field. You can use it but it may not
+      work as expected.
+    </AlertCard>
+  </Story>
+  <Story name="warning dark">
+    <AlertCard
+      onClose={() => undefined}
+      color={colors.orange.base}
+      icon={<IconWarningSolid />}
+      heading="Warning alert"
+      theme="dark"
+      actions={
+        <Button theme="dark" size="small">
+          Confirm
+        </Button>
+      }
+    >
+      exceededSubLimitAt is a deprecated field. You can use it but it may not
+      work as expected.
+    </AlertCard>
+  </Story>
+</Preview>
+
+## Error Alert
+
+Error alerts let the user know that something has gone wrong or is blocked in the given workflow they are trying to complete. When possible, error alerts should contain helpful information about what has happened to help unblock the user (such as steps to resolve or a link to a docs article).
+
+<Preview>
+  <Story name="error light">
+    <AlertCard
+      onClose={() => undefined}
+      color={colors.red.base}
+      icon={<IconErrorSolid />}
+      heading="Error alert"
+      style={{ marginBottom: 20 }}
+    >
+      Something is broken. You cannot perform this action at this time.
+    </AlertCard>
+  </Story>
+  <Story name="error dark">
+    <AlertCard
+      onClose={() => undefined}
+      color={colors.red.base}
+      icon={<IconErrorSolid css={{ "& .inner": { fill: colors.white } }} />}
+      heading="Error alert"
+      theme="dark"
+      style={{ marginBottom: 20 }}
+    >
+      Something is broken. You cannot perform this action at this time.
+    </AlertCard>
+  </Story>
+</Preview>
+
+## Success Alert
+
+Success alerts are confirmation that the action the user was trying to take has succeeded.
+
+<Preview>
+  <Story name="success light">
+    <AlertCard
+      onClose={() => undefined}
+      color={colors.green.base}
+      icon={<IconSuccessSolid />}
+      heading="Success alert"
+    >
+      A new schema registry has been created by timbotnik.
+    </AlertCard>
+  </Story>
+  <Story name="success dark">
+    <AlertCard
+      onClose={() => undefined}
+      color={colors.green.base}
+      icon={<IconSuccessSolid css={{ "& .inner": { fill: colors.white } }} />}
+      theme="dark"
+      heading="Success alert"
+    >
+      A new schema registry has been created by timbotnik.
+    </AlertCard>
+  </Story>
+</Preview>
```

Comment:
I missed this in my review, so I'm sorry. We're not going to share contextual components as Space Kit ideas, just components that can be configured. Instead of demonstrating like this, demonstrate how the component can be configured.

Reply:
I can definitely remove the concepts of error/warning/success/error.
before I do that, @jglovier I think you might have had thoughts on this?